static/app/views/settings/organizationMembers/organizationMemberDetail.tsx

```diff
@@ -370,6 +375,7 @@ class OrganizationMemberDetail extends AsyncView<Props, State> {
               onAddTeam={this.handleAddTeam}
               onRemoveTeam={this.handleRemoveTeam}
               loadingTeams={!initiallyLoaded}
+              enforceIdpProvisioned
```

Comment:
Nit: The order of props should be consistent to make it easier to read. `enforceXYZ` came first for `OrganizationRoleSelect`, it should be the same for `TeamSelect` too.